ui/desktop/app/styles/app.scss

```diff
@@ -653,6 +653,38 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
+// Host Modal
+.host-list-container {
+  border: 1px solid var(--token-color-border-primary);
+  padding: 0.25rem;
+  border-radius: 6px;
+}
+
+.host-container {
+  button {
```

Comment:
This is a design question but I wonder if there should be some interaction when hovering over the buttons? I don't get any feedback when hovering over them which feels like it doesn't make it obvious they're buttons

Reply:
That is a great feedback. Yeah I can tweak this to give it some indication that you are hovering over a button. I will move that to a new PR since this is already large.